trezor-crypto/src/bip32.c

```diff
@@ -475,6 +485,8 @@ void hdnode_fill_public_key(HDNode *node)
 		node->public_key[0] = 1;
 		if (node->curve == &ed25519_info) {
 			ed25519_publickey(node->private_key, node->public_key + 1);
+		} else if (node->curve == &ed25519_blake2b_nano_info) {
+			ed25519_publickey_blake2b(node->private_key, node->public_key + 1);
```

Comment:
Actually it should be 33 as well because you are skipping the first byte here.

Reply:
I think you might have mistaken the first argument here as `node->public_key` (instead of _private_key_). This should be be correct (otherwise my coin integration unit-tests should fail miserably).